config/options.yml

```diff
@@ -10,4 +10,4 @@ shared:
     google_auth_client_secret: <%= ENV["GOOGLE_AUTH_CLIENT_SECRET"] || Rails.application.credentials[:google_auth_client_secret] %>
     http_header_auth_email: <%= ENV["HTTP_HEADER_AUTH_EMAIL"] || "X-WEBAUTH-EMAIL" %>
     http_header_auth_name: <%= ENV["HTTP_HEADER_AUTH_NAME"] || "X-WEBAUTH-NAME" %>
-    http_header_auth_uid: <%= ENV["HTTP_HEADER_AUTH_UID"] || "X-WEBAUTH-USER" %>
```

Comment:
The thing is, when you google for `X-WEBAUTH-USER`, you will find examples which use this name. This is what I would stick with this instead of `-UID`